jobs/JEVS_NWPS_STATS

```diff
@@ -64,15 +64,18 @@ export ARCmodel=${COMIN}/prep/${COMPONENT}
 export COMOUT=${COMOUT:-$(compath.py -o ${NET}/${evs_ver})/${STEP}/${COMPONENT}}
 export COMOUTsmall=${COMOUT}/${RUN}.${VDATE}/${MODELNAME}/${VERIF_CASE}
 export COMOUTfinal=${COMOUT}/${MODELNAME}.${VDATE}
+export COMINhist=${COMINnwps}/${MODELNAME}.${VDATE}/nwges/dcom_hist.txt
```

Comment:
When NCO sees `COMINhist`, they'll think it is the com directory for a model name hist. Where is in the code it uses `COMINhist`, can it use `${COMINnwps}/${MODELNAME}.${VDATE}/nwges/dcom_hist.txt` instead?